### PR TITLE
Add SubscribeNow component and localization

### PR DESCRIPTION
**Pull request by guillermoscript**
This pull request adds the SubscribeNow component to the codebase. It also includes localization for the SubscribeNow component, allowing it to display different titles and descriptions based on the user's language.

### CODE REVIEW

File: actions/dashboard/AI/ShowCaseActions.tsx [hunk 1]
```diff
@@ -148,11 +149,14 @@ export async function continueShowCaseChatConversation(
...
-                            <SuccessMessage
-                                status="success"
-                                message="Assignment marked as completed."
-                                fire={true}
-                            />
+                            <div className="w-full flex flex-col gap-8">
+                                <SuccessMessage
+                                    status="success"
+                                    message="Assignment marked as completed."
+                                    fire={true}
```
Comment (HectorZR) [inline actions/dashboard/AI/ShowCaseActions.tsx:156]: If prop is true, no need for `fire={true}`. With just `fire` should work

**Review by HectorZR — COMMENTED** [review]
Just a little detail

File: actions/dashboard/AI/ShowCaseActions.tsx (hunk 1 above)
Comment (guillermoscript) [reply actions/dashboard/AI/ShowCaseActions.tsx:156]: jejej true XD

File: components/home/SubscribeNow.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```
Comment (angel-afonso) [inline components/home/SubscribeNow.tsx:1]: Importing react in 2024 (?)

(Just a joke xd )

**Pull request merged**